contents/blog/product-engineer-vs-software-engineer.md

```diff
@@ -14,71 +15,77 @@ crosspost:
+Product engineers also spend time talking to customers, digging into usage data, and researching the competitive landscape. They test other products, build prototypes, and brainstorm experiments to make their product better. They use this research to prioritize a roadmap that they own, and take responsibility for explaining the roadmap and gathering feedback from users and the company.
+
+Product engineering also tends to be more fast-paced, scrappy, and independent than software engineering. This is especially true as AI tools are on the rise; product engineers can move even quicker by using LLMs to handle less-critical implementation details.
+
+Software engineers code too, of course, but they also spend lots of time designing, testing, and maintaining what they build. They look for ways to optimize systems, improve scalability, and solve bugs. They are deeply focused on specific areas of technology, whether those are databases, data pipelines, backend APIs, or client-side app frameworks.
 
-Software engineers focus on writing code, testing it, and maintaining it. They look for ways to optimize code, improve scalability, and solve bugs. They are deeply focused on specific areas of technology, whether that is databases, data pipelines, backend APIs, or client-side app frameworks. This means researching and reading documentation, updates, and code.
+Software engineering projects also tend to have longer timelines and cross-functional collaboration because of their specialization.
 
 ## Why is demand for product engineers growing?
 
-There's no single reason, but typically it's motivated by a desire to ship product improvements faster. Product engineers are popular among startups and early-stage companies for this reason, even if they don’t say it. Many engineers behave like product engineers, even if it's not their job title.
+There's no single reason, but it's often simply because people want to ship faster. Product engineers are popular among startups and early-stage companies for this reason, even if they don’t say it. Many software engineers behave like product engineers, even if it's not their job title.
 
-At larger companies, product engineers are seen as a way to drive innovation, sometimes through the creation of smaller teams that have the freedom to ideate and execute product ideas. Software engineers and multidiscipline teams are still important, but product engineers can inject urgency where it's needed.
+For example, Linear, the wildly popular modern project management tool, is built [almost entirely by product engineers](https://linear.app/now/how-we-run-projects-at-linear). Engineers rotate through project leadership roles and talk directly to users through their beta program to make decisions based on user feedback rather than traditional metrics. 
```

Comment:
I think this is a good example, but I'd love to see the paragraph look like this:
1. Intro to Linear (same as you have now)
2. Some specific product engineering things they do (it's a bit unclear what "engineers rotate through project leadership roles" means/relates to and the second half of sentence is quite wordy). 
3. How this helps them ship faster